libs/CMakeLists.txt

```diff
@@ -3,4 +3,7 @@ add_library(toylib STATIC
                 toylib.hpp
                 toylib.cpp
 )
-target_include_directories(toylib PUBLIC .)
\ No newline at end of file
+target_include_directories(toylib PUBLIC $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/libs>)
```

Comment:
what happens here is that if you install the library, you can not include source directory in general (i.e. for install interface), only for the build interface